### DISCUSSION TITLE
Retaining parameter signature documentation

### DISCUSSION THREAD

**Discussion by sQVe** [folke/noice.nvim#421] [Q&A]
Hey 👋🏼,

I'm trying `noice` out and find it improving many things. There's one thing I'm noting that is keeping me from using it though, which is that it removes the documentation for parameters. See screenshorts:

When triggering `vim.lsp.buf.signature_help()` without `noice`. Note that we're getting the `- Bar` parameter documentation for `b`:

![2023-04-06_09-55](https://user-images.githubusercontent.com/2284724/230311878-eda8ba91-a9c6-4c95-945d-cb00cba8b519.png)

With `noice` there is no documentation for `b`:
![2023-04-06_09-28](https://user-images.githubusercontent.com/2284724/230312816-cf1e75dc-377d-48a8-9921-8c5d2624fa99.png)

Any help or input is greatly appreciated! 🙏🏼

**Comment by folke** [comment]
Fixed in https://github.com/folke/noice.nvim/commit/e76ae13dd272dc23d0154b93172d445aeabad8f1
